templates/components/listings/people/user.html.twig

```diff
@@ -2,12 +2,12 @@
     <div>
         <div class="not-sidebar">
             <div>
-                <h2 class="txt-mars">
+                <{{ headTag }} class="txt-mars">
```

Comment:
Instead of `headTag` could you use `heading` and set a default value.

FYI this is what we used in the main app (master branch) in `templates/People/index.html.twig` which has: `<{{ heading|default('h4') }}>`

Also if we are modifying the heading level I believe that the class should also be configurable.
Could it be turned to a parameter, and even better an optional parameter?

FYI in the case of https://github.com/w3c/w3c-website/issues/276 where we want the heading to be a `<h3>` level, then setting `class=txt-mars` is not needed as this is the [default size for that heading](https://design-system.w3.org/styles/typography.html#headings).

Overall this could look like:
```
<{{ heading|default('h3') class="heading_class|default('')"}}>
(...)
</{{ heading|default('h3') }}>
```

Reply:
addressed in https://github.com/w3c/w3c-website-templates-bundle/pull/83/commits/9c0548dd7af45a5651d95b6f46b6b11b9d0b84dc